frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -228,7 +256,13 @@ class Multiselect extends React.PureComponent<Props, State> {
             value={this.valueFromState}
             disabled={disabled}
             size={"compact"}
+            noResultsMsg={
+              this.props.element.maxSelections !== 0
+                ? this.getNoResultsMsg()
+                : "No results"
```

Comment:
I think it'd make more sense for all of this logic to live in the `getNoResultsMsg` helper function rather than have half of it there and half of it here